dbt/adapters/athena/impl.py

```diff
@@ -80,8 +102,9 @@ class AthenaConfig(AdapterConfig):
     native_drop: Optional[str] = None
     seed_by_insert: Optional[bool] = False
     lf_tags_config: Optional[Dict[str, Any]] = None
-
-
+    seed_s3_upload_args: Optional[Dict[str, Any]] = None
+    partitions_limit: Optional[int] = None
```

Comment:
fine for now, as it's future proff - but that params is not used at all, we must add it as fallback in the model configs.